frontend/src/pages/data-product/components/data-product-tabs/data-product-tabs.tsx

```diff
@@ -129,6 +157,68 @@ export function DataProductTabs({ dataProductId, isLoading }: Props) {
         ];
     }, [dataProductId, t, dataProductHistoryData, isFetchingDataProductHistory]);
 
+    const steps: TourProps['steps'] = useMemo(() => {
+        return [
```

Comment:
Maybe start with a step:
Congrats on creating your first data product, this tour will guide you in the next steps to making a great data product.

Reply:
This is just an idea